internal/api/lib_test.go

```diff
@@ -868,6 +869,8 @@ func Benchmark100ConcurrentContractCalls(b *testing.B) {
 		resChan := make(chan []byte, callCount)
 		wg.Add(callCount)
 
+		info = mockInfoBinNoAssert("fred")
```

Comment:
Why the new function just for this single benchmark? Let's keep using `MockInfoBin` here.